packs/bit-bucket/actions/delete_issue.yaml

```diff
@@ -0,0 +1,14 @@
+name: delete_issue
+runner_type: run-python
+description: Delete issue for a given repository
+enabled: true
+entry_point: delete_issue.py
+parameters:
+  repo:
+    type: string
+    description: provide repositroy name for listing the branches
+    required: true
+  id:
+    type: array
+    description: ID of the issue to delete
```

Comment:
Confused about this one. 

It looks like you pass in an array and not a single id. In this case, the action should be called `delete_issues` and the argument should be called `ids`. As an alternative, you can change the type to `string` and update the action so it operates with a single id and not a list.